src/lib/workspace/instruction-modal.ts

```
@@ -0,0 +1,17 @@
+  const safeUserId = userId ?? "anonymous";
+  return [
+    `workspace-instruction-new-workspace:${safeUserId}:${workspaceId}`,
+    `workspace-instruction-new-workspace:${workspaceId}`,
+  ];
+}
+
+export function markNewWorkspaceInstruction(userId: string | null | undefined, workspaceId: string): void {
+  if (!workspaceId) return;
+  const keys = getNewWorkspaceInstructionKeys(userId, workspaceId);
+  try {
+    keys.forEach((key) => window.localStorage.setItem(key, "true"));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)